modules/nw-sriov-hwol-supported-devices.adoc

```diff
@@ -22,4 +22,23 @@ Hardware offloading is supported on the following network interface controllers:
 |MT28880 Family [ConnectX&#8209;5{nbsp}Ex]
 |15b3
 |1019
-|===
\ No newline at end of file
+|===
+
+.Technology preview network interface controllers
+[cols="1,2,1,1"]
+|===
+|Manufacturer |Model |Vendor ID | Device ID
+
+|Mellanox 
+|MT2894 Family [ConnectX-6 Lx]
+|15b3
+|101f
+
+|Mellanox
+|MT42822 BlueField-2 in ConnectX-6 NIC mode
+|15b3
+|a2d6
+|===
+
+:FeatureName: ConnectX-6 Lx support and BlueField-2 in ConnectX-6 NIC mode support
```

Comment:
Per the SSG we usually try to avoid using "support" with tech preview features. 

> Never use the phrase "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", and so on.

Could be rephrased, maybe like:

```suggestion
:FeatureName: Using a ConnectX-6 Lx or BlueField-2 in ConnectX-6 NIC mode device
```